### ISSUE TITLE
Create tokens in code with Style Dictionary

### ISSUE THREAD

**Issue by vnys** [equinor/design-system#2737]
- [x] Decide on directory structure
- [x] #2791 
- [ ] Generate tokens for CSS[^1]
- [ ] Generate tokens for React[^2]

[^1]: Let’s start with CSS because it’s simpler, not necessarily something we will publish as of now
[^2]: What’s missing before we can use the new tokens in the button component?

**Comment by vnys** [comment]
## Meeting minutes 2023-05-16

Nina, Oddbjørn, Victor

- Try to remove component tokens to reduce complexity
- Add type value as second segment in tokens name (sometimes? always?)
- Start with button colour
- Export semantic tokens (CSS, JSON, Tailwind Config?)
- Create tokens table as replacement for component tokens

**Comment by vnys** [comment]
> This is too generic. It's not a task and not a user story.

I agree, it’s more an epic.
